shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1184,6 +948,9 @@ export default {
         // Store default versions
         this.defaultRke2 = defaultRke2;
         this.defaultK3s = defaultK3s;
+        this.versionsInfo = {
```

Comment:
This needs to be removed and the specific properties needed by `basics` passed through. that means the computed properties within `basics` can go away and the interface between parent and child containers is cleaner

Reply:
why do you think it is cleaner that way? If nothing else uses those properties